File: app/src/main/java/com/nimtego/itunes/model/Song.java
package com.nimtego.itunes.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class Song {

    private Integer artistId;
    private Integer collectionId;
    private Integer trackId;
    private String artistName;
    private String wrapperType;
    private String trackName;
    private Double trackPrice;
    private Integer trackTimeMillis;
}

File: app/src/main/java/com/nimtego/plectrum/presentation/mvp/model/song/Song.kt
package com.nimtego.plectrum.presentation.mvp.model.song

data class Song (
    val artistId: Int,
    val collectionId: Int,
    val trackId: Int,
    val artistName: String,
    val wrapperType: String,
    val trackName: String,
    val trackPrice: Double,
    val trackArtWorkUrl: String,
    val trackTimeMillis: Int)